Jenkinsfile

```diff
@@ -96,28 +96,44 @@ def buildGrakn() {
     sh "build-grakn.sh ${env.BRANCH_NAME}"
 }
 
-//Run all tests
-node {
-    String workspace = pwd()
-    checkout scm
-
-    slackGithub "Janus tests started"
-
-    timeout(120) {
-	stage('Run Janus test profile') {
-	    try {
-		sh "mvn clean verify -P janus -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT"
-	    } finally {
-		junit "**/TEST*.xml"
-	    }
-	}
-    }
+def shouldRunAllTests() {
+    // TODO bad
+    return true
+    return env.BRANCH_NAME in ['master', 'stable']
+}
 
-    slackGithub "Janus tests success", "good"
+def shouldDeployLongRunningInstance() {
+    return env.BRANCH_NAME == 'stable'
 }
 
-//Only run validation master/stable
-if (env.BRANCH_NAME in ['master', 'stable']) {
+// This is a map of jobs to perform in parallel, name -> job closure
+jobs = [
+        tests: {
+            //Run all tests
+            node {
+                String workspace = pwd()
+                checkout scm
+
+                slackGithub "Janus tests started"
+
+                timeout(120) {
+                    stage('Run Janus test profile') {
+                        try {
+                            sh "mvn clean verify -P janus -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT"
```

Comment:
`verify`. . . I know know why the Jenkins build was failing on my crazy test refactor PR.